scripts/protocgen.sh

```diff
@@ -4,14 +4,15 @@
 # docker build --pull --rm -f "contrib/devtools/Dockerfile" -t cosmossdk-proto:latest "contrib/devtools"
 # docker run --rm -v $(pwd):/workspace --workdir /workspace cosmossdk-proto sh ./scripts/protocgen.sh
 
-set -eo pipefail
```

Comment:
I think keeping `set -e` will be OK so that the script will exit if any command errors.